app/presenters/summary/components/property.rb

```diff
@@ -0,0 +1,95 @@
+module Summary
+  module Components
+    class Property < BaseRecord
+      alias property record
+
+      private
+
+      def answers # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+        attributes =
+          [
+            Components::FreeTextAnswer.new(
+              :house_type, house_type
+            ),
+            Components::FreeTextAnswer.new(
+              :bedrooms, property.bedrooms.to_s
+            ),
+            # TODO: Temporary fix to avoid duplicate keys in summary.yml
+            Components::MoneyAnswer.new(
+              :property_value, property.value
+            ),
+            Components::MoneyAnswer.new(
+              :outstanding_mortgage, property.outstanding_mortgage
+            ),
+            Components::PercentageAnswer.new(
+              :percentage_applicant_owned, property.percentage_applicant_owned
+            )
+          ]
+
+        # TODO: Crime application is nil error
+        if property.include_partner?
+          attributes << Components::PercentageAnswer.new(
+            :percentage_partner_owned, property.percentage_partner_owned
+          )
+        end
+
+        attributes << Components::ValueAnswer.new(:is_home_address, property.is_home_address)
+
+        if (property.is_home_address == YesNoAnswer::NO.to_s) && property.address&.values.present?
```

Comment:
This needs to handle` property.is_home_address` being `nil`

`if ((property.is_home_address == YesNoAnswer::NO.to_s) || (property.is_home_address == nil)) && property.address&.values.present?
`

Reply:
Or we could set `property.is_home_address ` to no if there is no home address recorded on 'client'